Gemfile

```diff
@@ -17,6 +17,8 @@ ruby_version_segments = Gem::Version.new(RUBY_VERSION.dup).segments
 minor_version = ruby_version_segments[0..1].join('.')
 
 group :development do
+  gem 'ed25519', '>= 1.2', '< 2.0'
```

Comment:
Should we add them to .sync.yaml file too

Reply:
Done